static/app/views/projectsDashboard/projectCard.tsx

```diff
@@ -103,7 +162,20 @@ class ProjectCard extends Component<Props> {
                 transactionStats={transactionStats}
               />
             </ChartContainer>
-            <Deploys project={project} />
+            <FooterWrapper>
+              <ScoreCardWrapper>
+                <ProjectStabilityScoreCard
+                  organization={organization}
+                  selection={projectCrashFreeRate}
+                  isProjectStabilized={isProjectStabilized}
+                  hasSessions={hasSessions}
+                />
```

Comment:
Reusing this component will cause firing an API request per each project card.
Seeing that this is inside an infinite list and some orgs have hundreds of projects, we should batch those calls together (same with fetchSessionsExistence).
If I am not mistaken those project requests are already batched.